### PR TITLE
Missing items section in homeroom schedule

### PR DESCRIPTION
**Pull request by vargaat**
refs: MBL-15476
affects: Student
release note: Add missing items to schedule view in homeroom.

test plan: See specification.

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/88083376/p/c8f4f47be8f90d62016ba54aae044151">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/88083385/p/c4f01de8a2c5372cfa4a4a2a756194e2">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/88083367/p/478b6676c250e9ad4c966590dfaf50f1">
    
  </a>
</details>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.84% | 91.94% | -0.1%

[MBL-15476](https://instructure.atlassian.net/browse/MBL-15476)

#### Affected Apps: Student

#### Release Note: 
Add missing items to schedule view in homeroom.
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Dashboard/K5/View/Schedule/K5ScheduleMissingItemsView.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct K5ScheduleMissingItemsView: View {
+    @Environment(\.appEnvironment) var env
+    @State private var isOpened: Bool = AppEnvironment.shared.userDefaults?.isMissingItemsSectionOpenOnK5Schedule ?? true
+    @State private var isArrowPointingDown = AppEnvironment.shared.userDefaults?.isMissingItemsSectionOpenOnK5Schedule ?? true
```
Comment (balintbartok) [inline Core/Core/Dashboard/K5/View/Schedule/K5ScheduleMissingItemsView.swift:24]: This feels a bit redundant, so I tried out removing it, at witnessed the chaos :) So I see why this is necessary..

**Review by balintbartok — APPROVED** [review]
QA + 1

**Pull request merged**